### ISSUE TITLE
loss is not decreasing at all

### ISSUE THREAD

**Issue by HAHA2XY** [yenchenlin/pix2pix-tensorflow#38]
I have trained hundreds of epochs and the loss of discriminator stay still at 1.38629460, loss of generator is also not decreasing. Samples given also shows no improvements at all. Could someone give me some suggestion?
